ci/kind/test-e2e-kind.sh

```diff
@@ -0,0 +1,59 @@
+#!/usr/bin/env bash
+
+# Copyright 2020 Antrea Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# This script is required for Antrea to work properly in a Kind cluster on Linux. It takes care of
+# disabling TX hardware checksum offload for the veth interface (in the host's network namespace) of
+# each Kind Node. This is required when using OVS in userspace mode. Refer to
+# https://github.com/vmware-tanzu/antrea/issues/14 for more information.
+
+# The script uses the antrea/ethtool Docker image (so that ethtool does not need to be installed on
+# the Linux host).
+
+set -eo pipefail
+
+TESTBED_CMD=$(dirname $0)"/kind-setup.sh"
+YML_CMD=$(dirname $0)"/../../hack/generate-manifest.sh"
+
+function quit {
+  if [[ $? != 0 ]]; then
+    echo " Test failed cleaning testbed"
+    $TESTBED_CMD destroy kind
+  fi
+}
+trap "quit" INT EXIT
+
+function run_test {
+  mode=$1
+  args=$2
+  echo "create test bed with args $(echo $args)"
+  eval "timeout 600 $TESTBED_CMD create kind --antrea-cni false $(echo $args)"
+  $YML_CMD --kind  --encap-mode $mode | docker exec -i kind-control-plane dd of=/root/antrea.yml
+  sleep 1
+  go test -v -timeout=20m github.com/vmware-tanzu/antrea/test/e2e -provider=kind
+  $TESTBED_CMD destroy kind
+}
+
+docker pull busybox
+docker pull nginx
+
+echo "======== Test encap mode =========="
+run_test encap '--images "busybox nginx antrea/antrea-ubuntu:latest"'
```

Comment:
maybe a stupid question, but why do we bother loading busybox & nginx into the Kind cluster, instead of letting the Nodes download it? Speed?

maybe define a bash variable instead of repeating the image list 3 times

Reply:
yes for speed, instead of loading images from internet on each node.  may not help much in the end,